metadata-ws/src/test/resources/features/reference-sequence.feature

```diff
@@ -218,8 +218,8 @@ Feature: reference sequence
       | name=NCBI36&patch=p2                            |
       | name=GRCh37&patch=p3                            |
       | name=GRCh38&patch=p3                            |
-      | accessions=GCA_000001405.2                      |
-      | name=GRCh37&patch=p3&accessions=GCA_000001405.3 |
+      | accession=GCA_000001405.2                       |
+      | name=GRCh37&patch=p3&accessio=GCA_000001405.3   |
```

Comment:
```suggestion
      | name=GRCh37&patch=p3&accession=GCA_000001405.3   |
```
how this passes ?

Reply:
My bad, fixed in https://github.com/EBIvariation/metadata-ws/pull/72/commits/b4713b4979914c92c985ebdcc016f0296cc07f62